cmd/juju/model/mocks/trackbranch_mock.go

```diff
@@ -15,6 +16,19 @@ type MockTrackBranchCommandAPI struct {
 	recorder *MockTrackBranchCommandAPIMockRecorder
 }
 
+// HasActiveBranch mocks base method
+func (m *MockTrackBranchCommandAPI) HasActiveBranch(arg0 string) (bool, error) {
```

Comment:
I wonder why this showed up :thinking:

Reply:
Oh, this was because  I updated mockgend in 2.7 but not in 2.6 during my last pr.
Thats why git thought he could do some wicked magic. I have fixed that. Good pointer